src/react/components/pages/prebuiltPredict/prebuiltPredictPage.tsx

```diff
@@ -816,4 +846,57 @@ export class PrebuiltPredictPage extends React.Component<IPrebuiltPredictPagePro
             }
         }
     }
+
+    private handleUpdateRequestURI = () => {
+        const {currentPrebuiltType, predictionEndpointUrl} = this.state;
```

Comment:
Add space in curly brackets, like `{ currentPrebuiltType, predictionEndpointUrl }`, or you can run your IDF's prettier to handle this kind of convention.